roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -40,7 +40,7 @@ class DockerImageAvailability(DockerHostMixin, OpenShiftCheck):
     # to look for images available remotely without waiting to pull them.
     dependencies = ["python-docker-py", "skopeo"]
     # command for checking if remote registries have an image, without docker pull
-    skopeo_command = "timeout 10 skopeo inspect --tls-verify={tls} {creds} docker://{registry}/{image}"
+    skopeo_command = "{envvars} timeout 10 skopeo inspect --tls-verify={tls} {creds} docker://{registry}/{image}"
```

Comment:
Please rename this to {proxyvars} as those are the only things we're actually setting.